### PR TITLE
OHRI-2087: Add unit tests for ohri programme summary tile in commons lib

### PR DESCRIPTION
**Pull request by lucyjemutai**
## Requirements

Add unit tests for ohri programme summary tile in commons lib

### CODE REVIEW

File: packages/esm-commons-lib/src/components/tile/ohri-programme-summary-tiles.test.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  it('should display the summary tiles as expected', () => {
+    render(<OHRIProgrammeSummaryTiles {...mockProps} />);
+    mockProps.tiles.forEach((tile) => {
+      expect(screen.getByText(tile.title)).toBeInTheDocument();
+      const subTitleElements = screen.getAllByText(tile.subTitle);
+      expect(subTitleElements.length).toBeGreaterThan(0);
+      const valueElements = screen.getAllByText(tile.value.toString());
+      expect(valueElements.length).toBeGreaterThan(0);
+    });
+  });
```
Comment (pirupius) [inline packages/esm-commons-lib/src/components/tile/ohri-programme-summary-tiles.test.tsx:28]: Please add one for tablet mode as well. There's another component with a different layout

**Review by pirupius — CHANGES_REQUESTED** [review]

File: packages/esm-commons-lib/src/components/tile/ohri-programme-summary-tiles.test.tsx (hunk 1 above)
Comment (lucyjemutai) [reply packages/esm-commons-lib/src/components/tile/ohri-programme-summary-tiles.test.tsx:38]: done

File: packages/esm-commons-lib/src/components/tile/ohri-programme-summary-tiles.test.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+// Mock data for OHRISummaryTileTablet
+const mockDetails = [
+  {
+    subTitle: 'Sample Subtitle',
+    value: 25,
+  },
+];
+
+describe('OHRIProgrammeSummaryTiles', () => {
+  it('should display the summary tiles as expected', () => {
```
Comment (pirupius) [inline packages/esm-commons-lib/src/components/tile/ohri-programme-summary-tiles.test.tsx:29]: Please add tests for loading state as well

**Review by pirupius — APPROVED** [review]
Thanks @lucyjemutai

**Pull request merged**